lib/v2/telegram/channel.js

```diff
@@ -127,15 +127,18 @@ module.exports = async (ctx) => {
                         let messageVideos = '';
                         if (item.find('.tgme_widget_message_video_player').length) {
                             item.find('.tgme_widget_message_video_player').each(function () {
-                                messageVideos += `<video src="${$(this).find('.tgme_widget_message_video').attr('src')}"
+                                const videoSource = $(this).find('.tgme_widget_message_video').attr('src');
+                                messageVideos += videoSource
+                                    ? `<video src="${videoSource}"
                             controls="controls" 
                             poster="${
                                 $(this)
                                     .find('.tgme_widget_message_video_thumb')
                                     .css('background-image')
                                     .match(/url\('(.*)'\)/)[1]
                             }"
-                            style="width: 100%"></video>`;
+                            style="width: 100%"></video>`
+                                    : '<b>Media is too big</b>';
```

Comment:
Please use [art-template](https://docs.rsshub.app/joinus/script-standard.html#template) for rendering HTML https://github.com/DIYgod/RSSHub/discussions/8359.